src/js/events.js

```diff
@@ -1,6 +1,26 @@
 (function () {
     'use strict';
 
+    function isElementDescendantOfExtension(extensions, element) {
+        return extensions.some(function (extension) {
+            if (typeof extension.getInteractionElements !== 'function') {
+                return false;
+            }
+
+            var extensionElements = extension.getInteractionElements();
+            if (!extensionElements) {
+                return false;
+            }
+
+            if (!Array.isArray(extensionElements)) {
```

Comment:
As far as I know, we've never supported IE8, and this is around for IE9 (which we still do support).  This is also not the first place we've used `Array.isArray()`.